evap/grades/templates/grades_upload_form.html

```diff
@@ -0,0 +1,25 @@
+{% extends "grades_course_base.html" %}
+
+{% load i18n %}
+
+{% block content %}
+    {{ block.super }}
+    
+    {% if final_grades %}
+        <div class="bs-callout bs-callout-info">
+            <b>{% trans "Automated publishing" %}</b><br />
+            {% blocktrans %}After this file with total grades was uploaded, the course will be published automatically once the evaluation ends. If the evaluation for this course has already ended, it will be published immediately. All contributors and participants will receive an email informing them about the published evaluation results and the available grade file document.{% endblocktrans %}
```

Comment:
is there a difference between total and final grades? any reason to not use the same word in code and UI?